sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3077,12 +3077,16 @@ object functions {
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
-   * Returns null if the array is null, true if the array contains `value`, and false otherwise.
+   * Returns null if the array is null, true if the array contains `value` or the content of
```

Comment:
We need to update this comment? I think `content of value` is a little ambiguous.

Reply:
Hey, thanks for the message. Do you want me to change the comment back? I see that you have started the test, is it too late?

Reply:
you can fix now